lib/policyEvaluator/utils/conditions.js

```diff
@@ -146,6 +146,7 @@ conditions.findConditionKey = (key, requestContext) => {
     map.set('s3:ObjLocationConstraint',
         headers['x-amz-meta-scal-location-constraint']);
     map.set('sts:ExternalId', requestContext.getRequesterExternalId());
+    map.set('keycloak:groups', requesterInfo.keycloakGroup);
```

Comment:
I'm not sure I understand what this does exactly?

Reply:
Short answer: as Arsenal is fixing the conditions keys internatlly, we want to add a new entry to support the `keycloak:groups` condition. This update will let Arsenal map the `keycloakGroup` field of the request context we create when we cant to evaluate a policy with the actual condition name, so, when the policy is being evaluated, the condition will be evaluated properly.

For more details, please refer to the original PR message.

Reply:
Added another condition mapping for the keycloak role. Even if we don't use it (depends on https://github.com/scality/citadel/pull/71), it will allow us to have this functionality in the future, if needed.

Reply:
Merging the PR as this is a blocker. I will reopen a new PR to remove the keycloak:roles condition if needed later.